static/css/thing.css

```diff
@@ -363,83 +363,6 @@
   padding-top: 1rem;
 }
 
-.media-modal-backdrop {
```

Comment:
Where did all this CSS go? I don't see it being added to another file.

Reply:
forgot to upload `app.css`. Done now.